### PR TITLE
dont export host env vars unless there is a config

### PR DESCRIPTION
**Pull request by salsal97**
provide explicit config to tests needing host envv

### CODE REVIEW

File: tests/myst/exec-not-signed/Makefile [hunk 1]
```diff
@@ -33,7 +33,6 @@ test-a:
 	grep -E "argv\[2]=green" result
 	grep -E "argv\[3]=blue" result
 	grep -E "argv\[4]=yellow" result
-	grep -E "TESTNAME=tests/myst/exec-not-signed" result
```
Comment (paulcallen) [inline tests/myst/exec-not-signed/Makefile:36]: We are expecting no environment variables, so create a result file for this test and then do a compare of the results to the expected output... which is probably just the args at this point

**Review by paulcallen — COMMENTED** [review]
Minor change in the test so we validate the non-existence of environment variables.
Other than that it looks good.

File: tests/myst/exec-not-signed/Makefile (hunk 1 above)
Comment (salsal97) [reply tests/myst/exec-not-signed/Makefile:36]: Added negative testing for this

**Review by mikbras — APPROVED** [review]

**Comment by mikbras** [comment]
Looks good. Thank you for doing this Saloni.

**Pull request merged**